android/java/res/layout/activity_brave_vpn_support.xml

```diff
@@ -178,7 +179,8 @@
             android:layout_marginEnd="16dp"
             android:layout_marginTop="16dp"
             android:layout_marginBottom="16dp"
-            android:background="@drawable/orange_rounded_button"
+            android:background="@drawable/rounded_holo_button"
```

Comment:
can we try `set_default_rounded_button_disabled.xml` if it looks good. because holo button still looks enabled.